jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -4,7 +4,7 @@
         # TODO(fejta): consider a stable tag instead of master
         git clone https://github.com/kubernetes/test-infra -b master
         './test-infra/jenkins/bootstrap.py' \
-            --bare \
+            '{soak-repos}' \
```

Comment:
@fejta, the soak-repos variable includes the --repo flag for both the repos and is assigned with this value '--repo=k8s.io/kubernetes --repo=k8s.io/release'
So actually it is gonna checkout 2 repos.